public/stylesheets/auth-splash.css

```diff
@@ -149,6 +153,9 @@
   display: block;
   margin: 2rem auto 0;
 }
+#demo-button {
+  margin-top: 10px;
```

Comment:
You're using rems in other places so please be sure to use rem here instead of pixels.